M2/BUILD/docker/README.md

```diff
@@ -21,6 +21,11 @@ The `Dockerfile` in this directory creates a container image based on latest Ubu
 1. Build a Macaulay2 container:
 ```
 make build
+```
+
+   Or, using the autotools build:
+```
+make build-autotools
```

Comment:
I don't think you should add this here because it's misleading. In `M2_BUILD_SCRIPT_autotools` you run `make` which does way more than just build M2-binary. The point of the docker system is that you can run M2 in such a way that packages from the clone are loaded and you can edit them and have that one run, so running the entire build process is extra and might cause confusion if files aren't loaded from the clone.

Reply:
Ok, that makes sense I think.  I've really only played around with these Dockerfiles to try building M2 in non-Debian-based Linux distros where I *do* want to run `make`to make sure that everything works.

Reply:
Yeah, I figured this is what you wanted. I don't think there's any harm on having many different directories with Docker/Makefiles specialized to different purposes, while the top directory minimal.